frontend/src/components/common/SectionBox.tsx

```diff
@@ -33,7 +33,7 @@ export function SectionBox(props: SectionBoxProps) {
 
   return (
     <>
-      {actualBackLink !== undefined && <BackLink to={actualBackLink} />}
+      {actualBackLink && <BackLink to={actualBackLink} />}
```

Comment:
The behavior will be different from before if you add this. Is this needed for the latest code to work?

Reply:
they should be functionally equivalent but it isn't necessary, reverting back